chat-ui/protogen.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+printf "Compiling protobuf definitions:\033[32;5m Cloudstate & Google\033[0m\n"
```

Comment:
ooh blinking, I like it

Reply:
Good catch, solid on MacOS but will blink on Linux. If you don't mind, I think it's better to make it solid green. ie. `\033[0;32m` as I am not sure blinking is popular.